includes.php

```diff
@@ -20,7 +20,7 @@
 		<link rel="stylesheet" href="index.css">
 	</head>
 	<body>
-		<h1>Patch demo</h1>';
+		<a href="."><h1>Patch demo</h1></a>';
```

Comment:
Shouldn't put block inside inline. Put the link inside the heading, then change the CSS to target `h1 > a` instead of `h1`. Remove the link colour and underline while there.

Reply:
For the record, this way is not incorrect though. `<a>` is not inline but "transparent", and it may contain anything, including block elements.

https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element
https://html.spec.whatwg.org/multipage/dom.html#transparent